db/templates.db.js

```diff
@@ -0,0 +1,97 @@
+const faker = require("faker");
+const productsTemplate = [...Array(40)].map((_) => {
+  return {
+    id: faker.datatype.uuid(),
+    name: faker.commerce.productName(),
+    description: faker.lorem.sentences(),
+    image: faker.random.image(),
+    price: faker.commerce.price(),
+    material: faker.commerce.productMaterial(),
+    brand: faker.lorem.word(),
+    inStock: faker.datatype.boolean(),
+    fastDelivery: faker.datatype.boolean(),
+    ratings: faker.random.arrayElement([1, 2, 3, 4, 5]),
+    offer: faker.random.arrayElement(["Save 50", "70% bonanza", "Prime"]),
+    productCategory: faker.random.arrayElement([
+      "tent",
+      "bags",
+      "shoes",
+      "ropes",
+      "jackets",
+      "gloves",
+    ]),
+    color: faker.commerce.color(),
+  };
+});
```

Comment:
- You can try to curate small amounts of data based on your theme instead of using `faker`.